content/docs/details/accessing-aws-resources/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Accessing AWS Services
+publishedDate: '2022-03-23T14:00:00.0Z'
+description: How to configure permissions so that Roadie can access AWS resources in your account.
+---
+## Prerequisites
+
+* The roadie account ID.
```

Comment:
This page is linked to from the kubernetes docs and S3 integration docs which explain how to get the ID. It's a bit awkward that the ID is supplied in the settings pages for integrations which require it (e.g. k8s, s3) rather than somewhere central.